tests/virt/node/migration_and_maintenance/test_workload_disruption_migration.py

```
@@ -0,0 +1,254 @@
+                }
+            }
+        }
+    ):
+        yield mode
+
+
+@pytest.fixture(scope="class")
+def vm_background_process_id(vm_with_hotplug_support):
+    if "windows" in vm_with_hotplug_support.name:
+        return start_and_fetch_processid_on_windows_vm(vm=vm_with_hotplug_support, process_name=REGEDIT_PROC_NAME)
+    else:
```

Ruff (0.15.7)

[warning] Unnecessary `else` after `return` statement

Remove unnecessary `else`

(RET505)